docs/docs/architecture.md

```diff
@@ -23,27 +23,26 @@ git clone git@github.com:Netflix/conductor.git
 
 #### Start Local Server
 
-The server is in the directory `conductor/server`.
+The server is in the directory `conductor/server`. To start it execute the following command in the root of the project.
 
 ```shell
-cd server
```

Comment:
Executing these commands gives the following error
```
FAILURE: Build failed with an exception.

* What went wrong:
Task 'server' not found in project ':conductor-server'.
```

I believe it makes sense to change the instructions to just `./gradlew bootRun`